cmd/gateway.go

```diff
@@ -0,0 +1,468 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+
+	"github.com/fi-ts/cloud-go/api/client/gateway"
+	"github.com/fi-ts/cloud-go/api/models"
+	"github.com/fi-ts/cloudctl/cmd/output"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"k8s.io/utils/pointer"
+)
+
+const (
+	flagProject                    = "project"
+	flagAllProjects                = "all-projects"
```

Comment:
I understand the idea of this flag. Problem is that we did it differently for the rest of the commands and we should try to be consistent within our CLI as much as possible. There is no explicit "all projects" flag in the other commands. It's the default. For filtering entity lists, there often is a `--project` and a  `--tenant` flag, which can be used to narrow down the result.